scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/language/json/JsonParser.scala

```diff
@@ -0,0 +1,60 @@
+package pl.touk.nussknacker.engine.language.json
+
+import cats.data.{NonEmptyList, Validated, ValidatedNel}
+import cats.data.Validated.{invalidNel, Valid}
+import io.circe.{parser, Json}
+import pl.touk.nussknacker.engine.api.Context
+import pl.touk.nussknacker.engine.api.context.ValidationContext
+import pl.touk.nussknacker.engine.api.expression.ExpressionTypingInfo
+import pl.touk.nussknacker.engine.api.generics.ExpressionParseError
+import pl.touk.nussknacker.engine.api.typed.typing.{TypingResult, Unknown}
+import pl.touk.nussknacker.engine.expression.parse.{CompiledExpression, ExpressionParser, TypedExpression}
+import pl.touk.nussknacker.engine.graph.expression.Expression
+import pl.touk.nussknacker.engine.graph.expression.Expression.Language
+import pl.touk.nussknacker.engine.spel.SpelExpressionParseError.JsonParsingError
+
+object JsonParser extends ExpressionParser {
+
+  override def languageId: Language = Expression.Language.Json
+
+  override def parse(
+      jsonString: String,
+      ctx: ValidationContext,
+      expectedType: TypingResult
+  ): ValidatedNel[ExpressionParseError, TypedExpression] = {
+    parseJson(jsonString).map { json =>
+      TypedExpression(
+        CompiledJsonExpression(jsonString, json),
+        JsonExpressionTypingInfo
+      )
+    }
+  }
+
+  override def parseWithoutContextValidation(
+      jsonString: String,
+      expectedType: TypingResult
+  ): ValidatedNel[ExpressionParseError, CompiledExpression] = {
+    parseJson(jsonString).map(json => CompiledJsonExpression(jsonString, json))
+  }
+
+  private def parseJson(jsonString: String): Validated[NonEmptyList[JsonParsingError], Json] =
```

Comment:
I thought that we would return Nussknacker's `Any`, I mean that we would do `ToJsonEncoder.encode` on parsed json. How does it work now? We expect somewhere that a value will be of type `Json`?